site/content/en/references/kustomize/kustomization/sortoptions/_index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "sortOptions"
+linkTitle: "sortOptions"
+type: docs
+weight: 23
```

Comment:
This should be 22, and the vars page should be updated to 23 (it is supposed to be alphabetical)